src/go/rpk/pkg/cli/cmd/cluster/config/import.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"reflect"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/out"
+	yaml "gopkg.in/yaml.v3"
+)
+
+func importConfig(
+	client *admin.AdminAPI,
+	filename string,
+	oldConfig admin.Config,
+	schema admin.ConfigSchema,
+	all bool,
+) (err error) {
+	readbackBytes, err := ioutil.ReadFile(filename)
+	out.MaybeDie(err, "Error reading file %s: %v", filename, err)
+	var readbackConfig admin.Config
+	err = yaml.Unmarshal(readbackBytes, &readbackConfig)
+	out.MaybeDie(err, "Error parsing edited config: %v", err)
+
+	type propertyDelta struct {
+		Property string
+		OldValue string
+		NewValue string
+	}
+	propertyDeltas := make([]propertyDelta, 0)
```

Comment:
nit: unless specifying a capacity, idiomatic Go uses `var propertyDeltas []propertyDelta` (no make).

Reply:
The `make` is used below for arrays that will go into the JSON request (where empty arrays should be `[]` and not `null`).  This convention becomes a bit awkward -- whether we construct arrays as nil or with `make` depends on whether they'll ultimately be serialized.

Reply:
Interesting. Sounds like from the redpanda side this array is non-nullable?